man/systemd-coredump.xml

```diff
@@ -77,18 +78,28 @@
     <citerefentry><refentrytitle>systemd.exec</refentrytitle><manvolnum>5</manvolnum></citerefentry>).
     </para>
 
-    <para><command>systemd-coredump</command> will log the coredump
-    including a backtrace if possible, and store the core (contents of
-    process' memory contents) in an external file on disk in
-    <filename>/var/lib/systemd/coredump</filename>, or directly in
-    the journal. This behavior may be modified using
-    <citerefentry><refentrytitle>coredump.conf</refentrytitle><manvolnum>5</manvolnum></citerefentry>.</para>
+    <para>The behaviour of <command>systemd-coredump</command> is configured through
+    <filename>/etc/systemd/coredump.conf</filename> and other configuration files. See
+    <citerefentry><refentrytitle>coredump.conf</refentrytitle><manvolnum>5</manvolnum></citerefentry>
+    for details. By default, <command>systemd-coredump</command> will log the coredump including a
+    backtrace if possible, and store the core (contents of process' memory contents) in an external
+    file on disk in <filename>/var/lib/systemd/coredump</filename>.</para>
 
-    <para>Apart from the
-    <citerefentry><refentrytitle>journalctl</refentrytitle><manvolnum>1</manvolnum></citerefentry>
-    log viewer,
+    <para>The log entry and a backtrace are stored in the journal, and can be viewed with
+    <citerefentry><refentrytitle>journalctl</refentrytitle><manvolnum>1</manvolnum></citerefentry>.
     <citerefentry><refentrytitle>coredumpctl</refentrytitle><manvolnum>1</manvolnum></citerefentry>
-    may be used to list and extract coredumps.</para>
+    may be used to list and extract coredumps or load them in
+    <citerefentry project='man-pages'><refentrytitle>gdb</refentrytitle><manvolnum>1</manvolnum></citerefentry>.
+    </para>
+
+    <para>The coredump helper is invoked anew each time. Therefore, any configuration
+    changes will take effect on the invocation of <command>systemd-coredump</command>.
+    If the sysctl configuration is modified, it must be updated in the kernel before
+    it takes effect, see
+    <citerefentry><refentrytitle>systemd-sysctl</refentrytitle><manvolnum>8</manvolnum></citerefentry>
+    and
+    <citerefentry project='man-pages'><refentrytitle>sysctl</refentrytitle><manvolnum>8</manvolnum></citerefentry>.
+    </para>
```

Comment:
Thanks!
I think we should mention a `systemd-coredump.socket` too.
See https://github.com/systemd/systemd/commit/3c171f0b1e